README.rst

```diff
@@ -25,6 +25,26 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
 This is easily achieved by downloading
 `the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
+Installing from PyPI
+====================
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-circuitpython-slideshow/>`_. To install for current user:
+
+.. code-block:: shell
+    pip3 install adafruit-circuitpython-slideshow
```

Comment:
Spacing here is incorrect, it's causing Sphinx to fail. Same for the code blocks following this. Please correct the spacing.